src/App.tsx

```diff
@@ -2,7 +2,14 @@ import { ApolloProvider } from '@apollo/client'
 import React from 'react'
 
 import { createApolloClient } from '@/api'
-import { ConnectionStatusProvider, OverlayManagerProvider, SnackbarProvider, StorageProvidersProvider } from '@/hooks'
+import {
+  useDialog,
```

Comment:
`useDialog` doesn't seem needed here?

Reply:
My bad but I think we should have eslint rule for that. WDYT?

Reply:
I've noticed that it's turned off in `.eslintrc`, what is the reason for that?

Reply:
We most certainly should have a rule for that! It's turned off because of some old dependencies we're locked in with because of shared `eslint-config`. We should get rid of that constraint soon